lib/oban.ex

```diff
@@ -372,21 +372,44 @@ defmodule Oban do
   end
 
   @doc """
-  Start a new supervised queue across all connected nodes.
+  Start a new supervised queue
 
-  ## Example
+  By default the new supervised queue will be across the connected nodes. Still,
+  if you prefer to start the queue only on the local node, you can pass the
+  option: `local_only: true`.
+
+  ## Examples
 
-  Start the `:priority` queue with a concurrency limit of 10.
+  Start the `:priority` queue with a concurrency limit of 10 across the
+  connected nodes.
 
       Oban.start_queue(:priority, 10)
       :ok
+
+  Start the `:media` queue with a concurrency limit of 5 only on the local node.
+
+      Oban.start_queue(:media, 5, local_only: true)
+      :ok
   """
   @doc since: "0.12.0"
-  @spec start_queue(name :: atom(), queue :: queue_name(), limit :: pos_integer()) :: :ok
-  def start_queue(name \\ __MODULE__, queue, limit) when is_queue(queue) and is_limit(limit) do
-    name
-    |> config()
-    |> Notifier.notify(:signal, %{action: :start, queue: queue, limit: limit})
+  @spec start_queue(
+          name :: atom(),
+          queue :: queue_name(),
+          limit :: pos_integer(),
+          opts :: Keyword.t()
+        ) :: :ok
+  def start_queue(name \\ __MODULE__, queue, limit, opts \\ [])
+      when is_queue(queue) and is_limit(limit) do
+    local_only? = Keyword.get(opts, :local_only, false)
+    payload = %{action: :start, queue: queue, limit: limit}
+
+    if local_only? do
+      send_signal(name, payload)
```

Comment:
* [ ] Add some unit tests for this case
* [ ] Check if the default path is already tested